### PR TITLE
Allow to edit recipients

### PR DESCRIPTION
**Pull request by jesusfcr**
Allow to edit the recipient emails in the team page

### CODE REVIEW

File: src/teams/home/home.vue [hunk 1]
```diff
@@ -19,6 +19,9 @@
                 <b-field label="Tag">
                   <b-input v-model="tag"></b-input>
                 </b-field>
+                <b-field label="Recipients">
+                  <b-input v-model="recipients" type="textarea" rows="5" placeholder="emails to receive notifications"></b-input>
```
Comment (manelmontilla) [inline src/teams/home/home.vue:23]: I would include brief instructions about the expected format of the list of emails. So I would
s/emails to receive notifications/list of emails to receive notifications (one per line)

File: src/teams/home/home.vue [hunk 2]
```diff
@@ -147,6 +157,9 @@ export default class Home extends Vue {
         this.name = team.name;
         this.tag = team.tag;
       }
+      const rec = this.recipients.split(/[ ,\n]+/).filter((e) => e.trim().length > 0);
```
Comment (manelmontilla) [inline src/teams/home/home.vue:160]: I would allow only one separator: `\n` in this case.

**Comment by manelmontilla** [comment]
Apart from some minor comments I would use the typed recipients API instead of the untyped one, I created a PR [here]( https://github.com/adevinta/vulcan-ui/pull/66) to modify the code to do so. If you agree you can just merge that PR into this one.

**Review by manelmontilla — CHANGES_REQUESTED** [review]

File: src/teams/home/home.vue (hunk 1 above)
Comment (jesusfcr) [reply src/teams/home/home.vue]: done

File: src/teams/home/home.vue (hunk 2 above)
Comment (jesusfcr) [reply src/teams/home/home.vue]: done, also I added email validation as the api accepts anything

**Review by manelmontilla — APPROVED** [review]
LGTM

**Pull request merged**